### PR TITLE
[Issue #24] Add translation draft for user-guide/annotations.md

### PR DESCRIPTION
**Pull request by sn00011**
See #24

### CODE REVIEW

File: docs/user-guide/annotations.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
 ---
 
-We have [labels](/docs/user-guide/labels) for identifying metadata.
 
-It is also useful to be able to attach arbitrary non-identifying metadata, for retrieval by API clients such as tools, libraries, etc. This information may be large, may be structured or unstructured, may include characters not permitted by labels, etc. Such information would not be used for object selection and therefore doesn't belong in labels.
+对于辨识metadata（元数据），我们有[labels](/docs/user-guide/labels)．
 
-Like labels, annotations are key-value maps.
+同时能够附件非辨识性的metadata也比较有用，可以被一些API的客户端如工具，库获取使用．这种信息可能会很大，可能是结构化的或者非结构化的，可能包含一些label不允许的字符等等．这些信息不会用来选择对象，因此不属于label的范畴．
```
Comment (resouer) [inline docs/user-guide/annotations.md]: 但是另一方面，我们也希望Kubernetes能够支持一种非辨识性的metadata，这些信息可以被一些API客户端包括工具包，依赖库等获取到。...

File: docs/user-guide/annotations.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+
+如Label一样，annotations是键值对的map．
```
Comment (resouer) [inline docs/user-guide/annotations.md]: annotations就是用来标示这些非辨识性元数据的。同Label一样，annotations的数据结构也是键值对map．

File: docs/user-guide/annotations.md [hunk 3]
```diff
@@ -14,14 +15,14 @@ Like labels, annotations are key-value maps.
...
-* fields managed by a declarative configuration layer, to distinguish them from client- and/or server-set default values and other auto-generated fields, fields set by auto-sizing/auto-scaling systems, etc., in order to facilitate merging
-* build/release/image information (timestamps, release ids, git branch, PR numbers, image hashes, registry address, etc.)
-* pointers to logging/monitoring/analytics/audit repos
-* client library/tool information (e.g. for debugging purposes -- name, version, build info)
-* other user and/or tool/system provenance info, such as URLs of related objects from other ecosystem components
-* lightweight rollout tool metadata (config and/or checkpoints)
-* phone/pager number(s) of person(s) responsible, or directory entry where that info could be found, such as a team website
+* 由一个声明式的配置层管理的字段，以和客户端或者服务端设置的默认值和其他自动生成的字段区分,也可能是其他的自动扩容/自动伸缩系统等等设置的值,这有利于合并
+* 构建/发布/镜像信息(时间戳,发布ID,git分支,Pull Request编号,镜像hash值,registry地址,等等)
+* 日志/监控/分析/审计的仓库指针
```
Comment (resouer) [inline docs/user-guide/annotations.md]: 日志、监控、分析、审计库的引用

File: docs/user-guide/annotations.md [hunk 4]
```diff
@@ -14,14 +15,14 @@ Like labels, annotations are key-value maps.
...
+* 客户端库/工具信息(例如为了调试目的 - 名称,版本,版本信息)
+* 其他的用户和/或者工具/系统的来源信息,如从其他生态系统来的组件的相关对象的URL
+* 轻量的rollout工具的元数据(配置或者检查点)
+* 相关负责人的电话/传呼号码,或者该信息能获取到的目录条目,如一个团队网站
 
-Yes, this information could be stored in an external database or directory, but that would make it much harder to produce shared client libraries and tools for deployment, management, introspection, etc.
\ No newline at end of file
+是的,这种信息可以保存在外部的数据库或者目录,但是这让创建用于部署,管理,视察的可共用的客户端库或者工具变得困难.
```
Comment (resouer) [inline docs/user-guide/annotations.md]: 总的来说，annotations的原理和用法跟Labels基本一致，只不过它负责代表的信息往往是用来被“程序”、“依赖库”，而不是用户获取和操作的。是的,这种信息也可以保存在 ....

**Comment by resouer** [comment]
Some comments from my side, thanks a lot.

**Comment by sn00011** [comment]
@resouer : I've made some changes based on your annotations.

**Comment by resouer** [comment]
Thanks, lgtm. Please squash your commits and I will merge this one :)

**Comment by hurf** [comment]
@resouer - if the branch is clean, now squash can be automatically done by choosing `Comfirm squash and merge` option after you pressed the big green button.

**Comment by resouer** [comment]
Great! Never know that it can be automatically squashed :)

**Pull request merged**